package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "scrypt-shim",
+  "version": "0.0.1",
+  "description": "scrypt shim",
+  "main": "src/index.js",
+  "browser": "src/browser.js",
+  "scripts": {
+    "postinstall": "node postinstall.js"
```

Comment:
You have the postinstall script in `src/`. So the path here needs to be adjusted to `src/postinstall.js` or maybe pull the script into the repo's root, or put it in `scripts/postinstall.js` (and adjust the path accordingly).